packages/rs-dapi-client/src/dapi_client.rs

```diff
@@ -227,7 +227,11 @@ impl DapiRequestExecutor for DapiClient {
                                     .address_list
                                     .write()
                                     .expect("can't get address list for write");
-
```

Comment:
We need to fix banning on retry.

The issue is that on dapi-client layer, StaleNode issue is not an error, so the banning logic is not executed.
It becomes an issue on Sdk layer, so we need to ban it there.

Reply:
TBD on separate PR